README.md

```diff
@@ -128,11 +128,15 @@ Now that we've installed SilverStripe, let's finely tune our development environ
 
 ### .env
 
-The main ingredient in environment management in SilverStripe is the `.env` file. This file provides a shared configuration across all your projects through the injection of environment variables. It should contain information such as database credentials, as those are most likely to be shared across all your projects. 
+The main ingredient in environment management in SilverStripe is the `.env` file. This file provides a shared configuration across all your projects through the injection of environment variables. It should contain information such as database credentials, as those are most likely to be shared across all of your local development projects. 
 
 It can also include other application settings. You might have API keys or email addresses in there that you want to specify as globally accessible by all projects. 
 
-The key attribute of the `.env` file, however, is that it does not have to ship with the project. It can live outside the web root, outside of source control. When you deploy this project from your local environment to somewhere else, that remote environment might have its own configuration, so having the file outside the project means you don't have to worry about overriding settings.
+The key attribute of the `.env` file, however, is that it should not ship with the project. It can live outside the web root, outside of source control. When you deploy this project from your local environment to somewhere else, that remote environment should have its own configuration, so having the file outside the project (or at least ignored by your source control system) means you don't have to worry about accidentally overriding settings.
+
+<div class="alert alert-danger">
```

Comment:
Typo: environments

So SS4 doesn't have any equivalent of \_ss_environment.php for production? That seems like a huge oversight to me.

Reply:
Fixed. And yeah, `.env` is it. But in production one should not be using it, instead opting to set environment variables via e.g. `SetEnv` in Apache, or the fast cgi set params directive for nginx, etc. Or just having them in the environment before starting the process, if the server only runs a single site.